### PR TITLE
added some more seed data

### PR DESCRIPTION
**Pull request by zaynshah**

### CODE REVIEW

File: seeds.sql [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
-INSERT INTO learning_objectives(cohort_id, topic, learning_objective) VALUES (1, 'React', 'Be able to handle events in React');
-INSERT INTO learning_objectives(cohort_id, topic, learning_objective) VALUES (1, 'React', 'Be able to use ternary operators in React');
-INSERT INTO learning_objectives(cohort_id, topic, learning_objective) VALUES (1, 'React', 'Be able to use conditional rendering');
+INSERT INTO learning_objectives(cohort_id, topic, learning_objective, not_confident) VALUES (1, 'HTML/CSS/Javascript', 'To know and contrast the roles of HTML/CSS and javascript','https://blog.hubspot.com/marketing/web-design-html-css-javascript#:~:text=HTML%20provides%20the%20basic%20structure,the%20behavior%20of%20different%20elements.');
+INSERT INTO learning_objectives(cohort_id, topic, learning_objective, not_confident, confident) VALUES (1, 'HTML/CSS/Javascript', 'Re-create a website using common HTML elements (h1, h2, ul, li, div...)','https://www.digitalocean.com/community/tutorial_series/how-to-build-a-website-with-html','https://www.proprofs.com/quiz-school/playquiz/?title=frontpage-quiz-2');
```
Comment (FahmidulHaque97) [inline seeds.sql:2]: Will be really useful for the presentation. Great work.

**Review by FahmidulHaque97 — APPROVED** [review]

File: seeds.sql [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
+INSERT INTO learning_objectives(cohort_id, topic, learning_objective, not_confident) VALUES (1, 'React', 'Use map to convert a JS object into a React component or HTML element','https://reactjs.org/docs/react-component.html');
+INSERT INTO learning_objectives(cohort_id, topic, learning_objective, not_confident) VALUES (1, 'SQL', 'To Understand and define what a relational database is','https://reactjs.org/docs/react-component.html');
+INSERT INTO learning_objectives(cohort_id, topic, learning_objective, not_confident) VALUES (1, 'SQL', 'Know how to write an SQL query to retrieve all data from a table ','https://reactjs.org/docs/react-component.html');
+INSERT INTO learning_objectives(cohort_id, topic, learning_objective, not_confident) VALUES (1, 'SQL', 'Be able to retrieve data from multiple table at the same time by using the join operator','https://reactjs.org/docs/react-component.html');
+INSERT INTO learning_objectives(cohort_id, topic, learning_objective, not_confident) VALUES (1, 'GIT', 'Know how to make a directory','https://reactjs.org/docs/react-component.html');
```
Comment (olliecase-green) [inline seeds.sql:25]: Nice addition of confident / not confident links

File: server.js [hunk 3]
```diff
@@ -131,8 +131,28 @@ async function getTopicsOnlyPerCohort(server) {
...
+  } catch (error) {
+    return false;
+  }
+  return true;
+}
+
 async function postLO(server) {
   const { cohort_id, topic, learning_objective, notConfident, confident } = await server.body;
+  console.log(confident);
+  console.log(notConfident);
```
Comment (olliecase-green) [inline server.js]: Have you considered removing console logs

**Review by olliecase-green — APPROVED** [review]
Looks good

**Pull request merged**